jovo-integrations/jovo-platform-googleassistant/src/core/GoogleActionResponse.ts

```diff
@@ -70,7 +70,7 @@ export class GoogleActionResponse implements JovoResponse {
         return true;
     }
 
-    hasSuggestionChips(...chips: string[]){
+    hasSuggestionChips(...chips: string[]): boolean {
```

Comment:
After adding documentation above, I noticed this was not added, so I added it to be consistent with other functions.